uint8 index # from 0
uint8 mode # 0: position model; 1: velocity mode; 2: torque(current) mode
float32 cmd

